src/cljc/athens/common_events/schema.cljc

```diff
@@ -114,6 +122,15 @@
      [:value string?]]]])
 
 
+(def datascript-unindent-multi
+  [:map
+   [:event/args
+    [:map
+     [:uids  vector?
+      :f-uid string?]]]])
```

Comment:
I don't know what to name this so this can be more useful because `f-uid` is 

```
[f-uid f-embed-id]          (common-db/uid-and-embed-id (first uids))
```

Can name this `first-uid` but that would raise the question if this is first uid then that can be extracted from `:uids` that is already passed.

Reply:
Name it `block-uid`.